### PR TITLE
Bugfix/arc 256 mobile navigation

### PR DESCRIPTION
**Pull request by BavoVdg**
- 'Bezoekersruimte' in hamburgermenu navigeert naar homepagina
- Sluit hamburger na navigatie
- Fix hover states in hamburgermenu

### CODE REVIEW

**Comment by bertyhell** [comment]
clicking on the "bezoekersruimtes" link in the hamburger when already on the homepage does not close the hamburger:
image

**Comment by bertyhell** [comment]
only the first case has a hover state, the others don't:
image

File: src/modules/navigation/components/Navigation/NavigationSection/NavigationSection.tsx [hunk 1]
```diff
@@ -21,6 +23,14 @@ const NavigationSection: FC<NavigationSectionProps> = ({
...
 	// Needed for overlay state. Dropdown state is saved in NavigationDropdown component
 	const [isHamburgerMenuOpen, setIsHamburgerMenuOpen] = useState(false);
+	const history = useSelector(selectHistory);
+
+	// Close dropdowns when the url path changed
+	useEffect(() => {
+		if (history?.[1] !== currentPath && isHamburgerMenuOpen) {
+			setIsHamburgerMenuOpen(false);
+		}
+	}, [history, currentPath, isHamburgerMenuOpen]);
```
Comment (bertyhell) [inline src/modules/navigation/components/Navigation/NavigationSection/NavigationSection.tsx:33]: Can't you handle this closing in the onClick handler of a navigation item or maybe in the onClick handler of the wrapper around the navigation items.
Or doesn't the event bubble to the top anymore?

**Review by bertyhell — CHANGES_REQUESTED** [review]

**Comment by BavoVdg** [comment]
@bertyhell 
- Added hover states
- Dropdown closes on click (handled by dropdown wrapper)

**Review by bertyhell — APPROVED** [review]

**Review by ian-emsens-shd — APPROVED** [review]

**Pull request merged**